test/suites/smoke/test-balances-consistency.ts

```diff
@@ -945,3 +1026,20 @@ describeSuite({
     });
   },
 });
+
+const AccountShortfalls = {
+  "Moonbase Stage": {
+    "0x7cb3790906f902a02f0d6784058362db929c5f3f": {
+      comment: "localAssetDeposit to be manually returned",
+      brokenIn: 2801,
```

Comment:
Should we change this to 2800?

Reply:
It's not broken **Since** but **In**.

The distinction being we want it to do the adjustment in RT2801 but to no longer apply in RT2900 (where hopefully we've applied this).